spec/rspec/rails/fixture_support_spec.rb

```diff
@@ -13,5 +13,15 @@ module RSpec::Rails
         expect(group).to respond_to(:fixture_path=)
       end
     end
+
+    it "doesn't raise a WrongScopeError on Rails 6.1" do
+      allow(ActiveRecord).to receive(:version) { Gem::Version.new('6.1.0') }
+
+      group = RSpec::Core::ExampleGroup.describe do
+        include FixtureSupport
+      end
+
+      expect { group.new.name }.to_not raise_error
```

Comment:
This doesn't really prove the issue, what code triggers the name usage?

Reply:
Issue that triggered the problem is when enrolling fixture transaction https://github.com/rails/rails/blob/b4ab1f19d8bcc4639c7379b560f000f01753b5b0/activerecord/lib/active_record/test_fixtures.rb#L101

This proves the issue as I just want to make sure `name` can be called without raising an error.
If you feel more confortable I can write an integration test and actually load fixtures, but this test should remain as I want to make sure `name` can be called within a spec

Reply:
This should be demonstrated by loading a fixtures, its an implementation detail that `name` is available as we never call it.